### DISCUSSION TITLE
Support Iterm2 image protocol for images

### DISCUSSION THREAD

**Discussion by VarLad** [rockorager/libvaxis#130] [General]
Kitty Graphics Protocol is much harder to implement than Iterm2 Image protocol, on the terminal side of things. 
Only Kitty itself apparently has a fully functioning implementation.
Wezterm also implements it but often bugged.

Even if there are no plans for Sixel, it would be nice if Iterm2 image protocol was supported.

**Comment by rockorager** [comment]
iTerm2 doesn't have any cropping or offsetting without re-uploading the image, which presents a similar performance issue that we would see with Sixel. I'm on the fence about it. I believe there are some use cases that it would be nice to have in the library, however there are some widgets I have planned for `vxfw` which would only work well with Kitty / block rendering...so that's where I am still debating on inclusion.

Also, iTerm2 is *only* supported by iTerm2 and Wez, but Wez also supports Kitty graphics.
